### PR TITLE
adds social icon button

### PR DESCRIPTION
**Pull request by rafibomb**

### CODE REVIEW

File: src/building-blocks/social-icon-button/social-icon-button.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<a href="#" class="social-icon-button twitter"><i class="fa fa-twitter" aria-hidden="true"></i><span></span></a>
+
+<a href="#" class="social-icon-button facebook"><i class="fa fa-facebook" aria-hidden="true"></i><span></span></a>
+
+<a href="#" class="social-icon-button gplus"><i class="fa fa-google-plus" aria-hidden="true"></i></i><span></span></a>
+
+
+
```
Comment (IamManchanda) [inline src/building-blocks/social-icon-button/social-icon-button.html]: Please remove these extra lines!

**Comment by rafibomb** [comment]
Done!

**Comment by rafibomb** [comment]
You wan tot merge it?

File: src/building-blocks/social-icon-button/social-icon-button.scss [hunk 2]
```diff
@@ -0,0 +1,81 @@
+$social-brand-facebook: #3b5998;
```
Comment (IamManchanda) [inline src/building-blocks/social-icon-button/social-icon-button.scss]: Please add the below comment on line 1 .... to let user customize if they want 

> `// Source: https://designpieces.com/2012/12/social-media-colours-hex-and-rgb/`

**Comment by IamManchanda** [comment]
ohk @rafibomb merging this .... but if for caring purpose will be great if you can add that comment later!

File: src/building-blocks/social-icon-button/social-icon-button.html [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<a href="#" class="social-icon-button twitter"><i class="fa fa-twitter" aria-hidden="true"></i><span></span></a>
+
+<a href="#" class="social-icon-button facebook"><i class="fa fa-facebook" aria-hidden="true"></i><span></span></a>
+
+<a href="#" class="social-icon-button gplus"><i class="fa fa-google-plus" aria-hidden="true"></i></i><span></span></a>
```
Comment (Deckluhm) [inline src/building-blocks/social-icon-button/social-icon-button.html]: As @blynnc reported in https://github.com/zurb/building-blocks/issues/227, there is a double `</i>` here.

**Comment by rafibomb** [comment]
Thanks @Deckluhm ! Are you able to make a Pull Request to fix it or open a new issue?

**Comment by Deckluhm** [comment]
Maybe @blynnc (who reported the issue) wants to open the PR to fix this? If not, I will handle it.

**Pull request merged**